a newly created keyspace sometimes won't get initialized

jbryan opened the pull request:
if you call delegate.drop() followed by delegate.create(), the pool will be unitialized.

Commit:
Made keyspace and system_manager available publicly

Commit:
Fixed bug where a newly created pool wouldn't get initialized

turtlebender commented:
@jbryan I get this parse error

``` pythonstack
 ======================================================================
ERROR: testParse (agamemnon.tests.test_rdf.GraphMemoryTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/trhowe/src/globusonline/agamemnon/agamemnon/tests/test_rdf.py", line 460, in testParse
    self.graph1.serialize("serialized.rdf")
  File "/Users/trhowe/src/globusonline/agamemnon/vendor/lib/python2.7/site-packages/rdflib/graph.py", line 781, in serialize
    serializer.serialize(stream, base=base, encoding=encoding, **args)
  File "/Users/trhowe/src/globusonline/agamemnon/vendor/lib/python2.7/site-packages/rdflib/plugins/serializers/rdfxml.py", line 49, in serialize
    write('<?xml version="1.0" encoding="%s"?>\n' % self.encoding)
  File "/Users/trhowe/src/globusonline/agamemnon/vendor/lib/python2.7/site-packages/rdflib/plugins/serializers/rdfxml.py", line 46, in <lambda>
    self.write = write = lambda uni: stream.write(uni.encode(encoding, 'replace'))
IOError: File not open for writing
```

turtlebender committed:
Updating requirements.txt and setup.py

Added version numbers and the correct dependencies in requirements.txt
and setup.py
